### PR TITLE
Sending Third Day challenges to review.

### PR DESCRIPTION
**Pull request by UnCarro**

### CODE REVIEW

**Review by edbermeo — CHANGES_REQUESTED** [review]
Add a .gitignore and exclude anything other than the code

File: Marco/Day03/hackerrank/Main.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package hackerrank;
+
+import hackerrank.solutions.*;
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/Main.java:3]: avoid the wildcard import

File: Marco/Day03/challenge/src/challenge/Main.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package challenge;
+
+import challenge.classes.*;
```
Comment (edbermeo) [inline Marco/Day03/challenge/src/challenge/Main.java:3]: Avoid the wildcard import

File: Marco/Day03/hackerrank/solutions/Two.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package hackerrank.solutions;
+
+class Arithmetic{
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Two.java:3]: Use a file per class

File: Marco/Day03/hackerrank/solutions/Two.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+    int add(int firstValue, int secondValue){
+
+        return firstValue + secondValue;
+
+    }
+}
+
+class Adder extends Arithmetic{
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Two.java:12]: Use a file per class

File: Marco/Day03/hackerrank/solutions/Two.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+}
+
+public class Two{
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Two.java:16]: Use a file per class

File: Marco/Day03/hackerrank/solutions/Third.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+import java.util.Scanner;
+
+public class Third {
+
+    public static void solving() {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("================================");
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Third.java:11]: Avoid magic strings

File: Marco/Day03/hackerrank/solutions/Third.java [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+        for(int i = 0; i < 3; i++)
+        {
+            String formatedString=scanner.next();
+            int formatedNumber=scanner.nextInt();
+
+            System.out.printf("%-15s%03d%n", formatedString, formatedNumber);
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Third.java:18]: Avoid magic strings

File: Marco/Day03/hackerrank/solutions/Bird.java [hunk 8]
```diff
@@ -0,0 +1,23 @@
+package hackerrank.solutions;
+
+class Animal {
+    void walk() {
+        System.out.println("I am walking");
+    }
+}
+
+public class Bird extends Animal {
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Bird.java:9]: Use a file per class

File: Marco/Day03/hackerrank/solutions/Bird.java [hunk 9]
```diff
@@ -0,0 +1,23 @@
+package hackerrank.solutions;
+
+class Animal {
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Bird.java:3]: Use a file per class

File: Marco/Day03/hackerrank/solutions/Third.java [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+        }
+
+        System.out.println("================================");
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Third.java:22]: Avoid magic strings

**Review by edbermeo — CHANGES_REQUESTED** [review]

File: Marco/Day03/hackerrank/solutions/Third.java [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+public class Third {
+
+    public static void solving() {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("================================");
+
+        for(int i = 0; i < 3; i++)
```
Comment (luisod78) [inline Marco/Day03/hackerrank/solutions/Third.java:13]: Even when all programming books teach us for loops with var names like i, j, x, y, etc. please avoid these. All the time try to name your variables as descriptive as possible

**Review by luisod78 — CHANGES_REQUESTED** [review]

File: Marco/Day03/hackerrank/solutions/Third.java [hunk 12]
```diff
@@ -7,19 +7,21 @@
     public static void solving() {
 
         Scanner scanner = new Scanner(System.in);
+        int numOfIterations = 3;
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Third.java:10]: Declare as a constant

File: Marco/Day03/hackerrank/solutions/Third.java [hunk 13]
```diff
@@ -7,19 +7,21 @@
...
 
-        System.out.println("================================");
+        String decorator = "================================";
```
Comment (edbermeo) [inline Marco/Day03/hackerrank/solutions/Third.java:12]: Declare as a constant

**Review by edbermeo — CHANGES_REQUESTED** [review]

**Review by edbermeo — APPROVED** [review]
LGTM

**Review by luisalanmz — APPROVED** [review]
LGTM

**Comment by UnCarro** [comment]
Thx!

**Pull request merged**